src/schema/notifications.ts

```diff
@@ -41,6 +41,34 @@ function findDataId(
 	)[0];
 }
 
+function addNewContactGroupToList(array: any, item: any, sortBy: any) {
+	if (sortBy === 'nameAsc') {
+		for (let i = 0; i < array.length; i++) {
+			const comparedResult = item.fileAsStr.localeCompare(
+				array[i].fileAsStr,
+				undefined,
+				{ sensitivity: 'base' }
+			);
+			if (comparedResult === -1) {
+				array.splice(i, 0, item);
+				return array;
+			} else if (i === array.length - 1) {
+				array.push(item);
+				return array;
+			}
+		}
+	} else {
+		array = [item].concat(array);
+		return array;
```

Comment:
Can we have single `return array` in this method?

Reply:
done